ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource

```diff
@@ -206,13 +206,18 @@ Click Minus Button
 Query Model With GRPCURL
     [Arguments]    ${host}    ${port}    ${endpoint}    ${json_body}
     ...            ${json_header}=${NONE}    ${insecure}=${FALSE}    ${plaintext}=${FALSE}
-    ...            ${background}=${NONE}    ${skip_res_json}=${FALSE}    &{args}
+    ...            ${background}=${NONE}    ${skip_res_json}=${FALSE}    ${cabundle_file}=${False}    &{args}
```

Comment:
can we use the same arg name as for the other keyword?

Reply:
the other keyword uses `verify` because it's a built-in of the requests library, I wouldn't want to mix the two to be honest

Reply:
I was referring to `cert` vs `cabundle_file`

Reply:
Done